_episodes/01-run-quit.md

```diff
@@ -5,55 +5,153 @@ exercises: 0
+
+~~~
+$ jupyter lab
+~~~
+{: .bash}
+
+### Windows Users
+To start the JupyterLab server you will need to access the command line through the Command 
+Prompt. Use the shortcut keys <kbd>Windows Logo Key</kbd> + <kbd>R</kbd> to launch the Run window, 
+then type in `cmd` and press Enter.
+
+After you have launched the Command Prompt, type the command:
+
+~~~
+$ jupyter lab
+~~~
+{: .bash}
+
+Below is a screenshot of a similar JupyterLab landing page to the one that should open in your 
+default web browser after starting the JupyterLab server on wither Mac OS X or Windows.
+
+<center>
+    <img alt="JupyterLab landing page" src="../fig/0_jupyterlab_landing_page.png" width="750"/>
+</center>
+
+## The JupyterLab Interface
+
+JupyterLab provides flexible building blocks for interactive, exploratory computing. While 
+JupyterLab has many features found in traditional integrated development environments (IDEs), it 
+remains focused on interactive, exploratory computing.
+
+The [JupyterLab Interface](https://jupyterlab.readthedocs.io/en/stable/user/interface.html) 
+consists of the Menu Bar, a collapsable Left Side Bar, and the Main Work Area which contains tabs 
+of documents and activities.
+
+### Menu Bar
+
+The Menu Bar at the top of JupyterLab has the top-level menus that expose various actions 
+available in JupyterLab along with their keyboard shortcuts (where applicable). The following 
+menus are included by default.
+
+*   **File:** Actions related to files and directories such as *New*, *Open*, *Close*, *Save*, etc. The *File* menu also includes the *Quit* action used to shutdown the JupyterLab server.
+*   **Edit:** Actions related to editing documents and other activities such as *Undo*, *Cut*, *Copy*, *Paste*, etc.
+*   **View:** Actions that alter the appearance of JupyterLab.
+*   **Run:** Actions for running code in different activities such as notebooks and code consoles (discussed below).
+*   **Kernel:** Actions for managing kernels which, as mentioned above, are separate processes for running code.
+*   **Tabs:** A list of the open documents and activities in the dock panel.
+*   **Settings:** Common JupyterLab settings can be configured using this menu. There is also an *Advanced Settings Editor* option in the dropdown menu that provides more fine-grained control of JupyterLab settings and configuration options.
+*   **Help:** A list of JupyterLab and kernel help links.
+
+A screenshot of the default Menu Bar is provided below.
+
+<center>
+    <img alt="JupyterLab Menu Bar" src="../fig/0_jupyterlab_menu_bar.png" width="750"/>
+</center>
+
+### Left Sidebar
+
+The left sidebar contains a number of commonly-used tabs, such as a file browser (showing the 
+contents of the directory in which the JupyterLab server was launched!), a list of running kernels 
+and terminals, the command palette, and a list of open tabs in the main work area. A screenshot of 
+the default Left Side Bar is provided below.
+
+<center>
```

Comment:
We should probably replace all instances of these with `<p align='center'>...</p>` since `<center>` is now obsolete: https://developer.mozilla.org/en-US/docs/Web/HTML/Element/center

See also https://gist.github.com/DavidWells/7d2e0e1bc78f4ac59a123ddf8b74932d

Reply:
Thanks for bringing this tom my attention. Didn't realize that `<center>` was obsolete.